roles/hostapd/tasks/deploy_configs.yml

```diff
@@ -6,8 +6,21 @@
     dest: '/etc/hostapd/hostapd_{{ item.name }}.conf'
     backup: yes
 
-- name: Add Default Config for hostapd
-  copy:
-    src: hostapd_default.conf
-    dest: '/etc/hostapd/hostapd.conf'
+- name: Show vlan hostapd configuration if vlan used
+  debug:
```

Comment:
```suggestion
  debug:
    verbosity: 1
```

Reply:
According to [debug module documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/debug_module.html), it will appear only if you run `ansible-playbook -v` (one -v)

Reply:
The main idea was to give some visible info for a quick review of the configuration in the context of using hostapd dynamic vlan.
If it comes to use ansible verbosity, then I am not sure it is useful, isn't it?

Reply:
As you prefer. I see two options:
- we keep as is: display information if necessary
- using verbosity: display information only if you specify `-v` at ansible execution

Second option is a best practice I've seen in some Ansible official document

Reply:
Ok than lets do it that way.
Thanks